### PR TITLE
Add timing to hash announcements

### PR DESCRIPTION
**Pull request by jobevers**
This could potentially be a performance issue
on reflector or any daemon with a large number of blobs.

### CODE REVIEW

File: lbrynet/core/server/DHTHashAnnouncer.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import binascii
 import collections
 import logging
+import time
```
Comment (jackrobison) [inline lbrynet/core/server/DHTHashAnnouncer.py]: should be imported out of`lbrynet.core.utils`

Comment (jobevers) [inline lbrynet/core/server/DHTHashAnnouncer.py]: will make this a separate PR to fix this in multiple files.

**Comment by jobevers** [comment]
https://github.com/lbryio/lbry/pull/339
